packages/core/src/tasks/taskHelpers.ts

```diff
@@ -28,7 +28,7 @@ export const selectPlatformIfRequired = async (
                     `Task "${knownTaskInstance?.task}" has only one supported platform: "${platforms[0]}". Automatically selecting it.`
                 );
                 c.platform = platforms[0];
-                c.program.opts().platform = c.platform;
+                // c.program.opts().platform = c.platform; not working currently
```

Comment:
Haven't tested myself, but I would say if we remove this with the PR, then it should be clarified on the channel why does it not work and what's the exact purpose of it